grafana-plugin/src/containers/DefaultPageLayout/DefaultPageLayout.tsx

```diff
@@ -1,11 +1,11 @@
 import plugin from '../../../package.json'; // eslint-disable-line
 import React, { FC, useEffect, useState, useCallback } from 'react';
 
-import { AppRootProps } from '@grafana/data';
 import { getLocationSrv } from '@grafana/runtime';
 import { Alert } from '@grafana/ui';
 import cn from 'classnames/bind';
 import { observer } from 'mobx-react';
+import { AppRootProps } from 'types';
```

Comment:
use the new strongly typed `AppRootProps`. See `types.ts` for more details, in short:
```ts
export type OnCallPluginMetaJSONData = {
  stackId: number;
  orgId: number;
  onCallApiUrl: string;
  license: string;
};

export type AppRootProps = BaseAppRootProps<OnCallPluginMetaJSONData>;
```